davenmathews/Bright_Events
# #153210393 Implement Password Reset

## This pull request implements password reset

This is on top of other previous commits

**The tasks completed are**
POST /api/v1/auth/reset_password

**This can be manually tested by:**
1. Clone the repository.
2. Use virtualenv to start a virtual environment.
3. Ensure the users.json and events.json are placed in your virtualenv folder and that they are empty
3. Install the requirements.txt.
4. Run appv1.py and open the local host in your browser.
5. Register a new user and log in.
6. Logout and reset their password to a new password of your choice. You should be redirected to the login page to log in with your new password.

**Constrains:**
User cannot reset password if logged in. It is assumed since they are logged in, they know the password.

**It Finishes and Delivers #153210393 story on PT**
